requirements.txt

```diff
@@ -1,2 +1,2 @@
 Scrapy==2.11.0
-zyte-spider-templates==0.3.0
+zyte-spider-templates==0.4.0
```

Comment:
Thanks for the reminder @Gallaecio !

We should only merge this when 'tis released.